### ISSUE TITLE
pascal-case-construct-id / no-construct-stack-suffix: template literal IDs bypass validation

### ISSUE THREAD

**Issue by ren-yamanashi** [ren-yamanashi/eslint-cdk-plugin#545]
### Describe the bug

`pascal-case-construct-id` and `no-construct-stack-suffix` validate the ID only when the second argument is a string `Literal`. An expression-less template literal (`` `my_bucket` ``) is skipped, although the rest of the suite explicitly treats expression-less template literals as static string IDs (`no-variable-construct-id` accepts them as valid static IDs; `prevent-construct-id-collision` validates them). Switching quotes to backticks therefore silently bypasses these naming checks.

**Repro:**

```ts
import { Bucket } from "aws-cdk-lib/aws-s3";
import { Construct } from "constructs";

export class PascalBasic extends Construct {
  constructor(scope: Construct, id: string) {
    super(scope, id);
    new Bucket(this, "control_bucket");  // control: reported
    new Bucket(this, `template_bucket`); // not reported
  }
}

export class SuffixDefault extends Construct {
  constructor(scope: Construct, id: string) {
    super(scope, id);
    new Bucket(this, "SampleBucketConstruct");  // control: reported
    new Bucket(this, `TemplateBucketConstruct`); // not reported
  }
}
```

### Expected Behavior

Expression-less template literal IDs are static strings and should be validated exactly like string literals (same as `prevent-construct-id-collision` already does via `quasis`).

### Current Behavior

Only the string-literal controls are reported:

```
repro/pascal-case-construct-id/basic.ts            8:22  error  Construct ID must be PascalCase  awscdk/pascal-case-construct-id
repro/no-construct-stack-suffix/default/suffix.ts  8:22  error  Construct ID 'SampleBucketConstruct' should not include Construct suffix  awscdk/no-construct-stack-suffix
```

Oxlint reports the identical set.

### Plugin

eslint-plugin-awscdk, oxlint-plugin-awscdk

### Plugin Version

eslint-plugin-awscdk 4.4.0, oxlint-plugin-awscdk 1.0.0

### Language Version

TypeScript 6.0.3 (eslint-plugin-awscdk) / TypeScript 7 preview via oxlint-tsgolint (oxlint-plugin-awscdk)

### Additional Context

- Verified on `main` (35b889a), rule-isolated configs.
- Sources: `packages/*/src/rules/pascal-case-construct-id.ts` and `packages/*/src/rules/no-construct-stack-suffix.ts` (`secondArg.type !== Literal` → return); compare `packages/*/src/rules/prevent-construct-id-collision.ts`, which handles `TemplateLiteral` with empty `expressions` via `quasis`.
- `no-parent-name-construct-id-match` has the same `Literal`-only check.
